File: codechef/Chef_and_Strings.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    long int test_cases;
    cin>>test_cases;
    while(test_cases--){
        long int n;
        cin>>n;
        long int sum=0;
        long int f,s;
        cin>>f;
        for(long int i=1;i<n;i++){
            cin>>s;
            sum+=abs(f-s)-1;
        
            f=s;
        }
        cout<<sum<<"\n";
    }
}

File: codechef/Chef_and_Strings.py
test=int(input())
while test>0:
    n=int(input())
    sum=0
    a=[map(int,input().split())]
    f=a[0]
    for i in a[1:]:
        sum+=abs(f-i)-1
        f=i
    print(f)
    test-=1